Samples/BooksQL/ViewModels/MutationViewModel.cs

```diff
@@ -0,0 +1,83 @@
+using System;
+using System.ComponentModel;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+using BooksQL.Models;
+using BooksQL.Services;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Xablu.WebApiClient.Services.GraphQL;
+using Xamarin.Forms;
+
+namespace BooksQL.ViewModels
+{
+    public class MutationViewModel : INotifyPropertyChanged
+    {
+        private BooksService _booksService;
+        private string _query = "Query result";
+        private string _result;
+
+        public MutationViewModel()
+        {
+            _booksService = DependencyService.Resolve<BooksService>();
+            RefreshCommand = new Command(() => CreateMutation());
+        }
+
+        public Command RefreshCommand { get; private set; }
+
+        public string Query
+        {
+            get { return _query; }
+            set
+            {
+                if (_query != value)
+                {
+                    _query = value;
+                    OnPropertyChanged();
+                }
+            }
+        }
+
+        public string Result
+        {
+            get { return _result; }
+            set
+            {
+                if (_result != value)
+                {
+                    _result = value;
+                    OnPropertyChanged();
+                }
+            }
+        }
+
+        private async Task CreateMutation()
+        {
+            var bookreview = new BookReview
+            {
+                BookISBN = "0544272994",
+                Review = "This is a mutation test"
+            };
+            var result = new MutationRequest<BookReview>(new MutationDetail("createReview", "review"), bookreview);
```

Comment:
This `MutationRequest` object should be passed as a parameter to the method call `_booksService.CreateReview()`. The method is doing the same thing internally